### ISSUE TITLE
.map expressions directly under a Fragment are being ignored

### ISSUE THREAD

**Issue by GiyoMoon** [solidjs-community/eslint-plugin-solid#106]
**Describe the bug**
The plugin doesn't warn about using `.map` on a signal if the expression is directly under a fragment in the jsx.

**To Reproduce**
```
git clone https://github.com/GiyoMoon/solidjs-eslint-map-bug.git
cd solidjs-eslint-map-bug
pnpm install
```
Open the `./src/App.tsx` file in your IDE which is running eslint. You should notice that only one of the two `.map` expressions are being noticed by eslint.

**Expected behavior**
Both `.map` expression should give eslint warnings.

**Screenshots**
![image](https://github.com/solidjs-community/eslint-plugin-solid/assets/36984337/c1e317ee-9146-4976-b72b-f0f054014b6c)

**Environment (please complete the following information):**

- OS: MacOS Venture 13.5
- Node version (`node --version`): v18.17.0
- `eslint-plugin-solid` version (`npm list eslint-plugin-solid`/`yarn why eslint-plugin-solid`): 0.13.0
- `eslint` version (`npm list eslint`/`yarn why eslint`): 8.47.0

- [ ] I would be willing to contribute a PR to fix this issue

**Comment by joshwilsonvu** [comment]
Thanks for catching this! You're right, the problem isn't caught directly under a Fragment, just regular JSX elements. They're parsed differently by ESLint, and I didn't catch that case. 

Will be a small fix—I'll have it out on the next release!

**Comment by GiyoMoon** [comment]
Thank you very much!
